tools/reporter/NswgReporter.js

```diff
@@ -85,6 +84,10 @@ class NswgReporter {
     }
   }
 
+  getCVEs () {
+    return this.report.data.attributes.cve_ids
```

Comment:
I'm not sure what the value will be if there is on cve_ids present, if it's an array still then that's great and we can just take the field value otherwise if it's null or something I'd do something like:
```
    return this.report.data.attributes.cve_ids || []
```

Reply:
My understanding is that it's an array, based on the example at https://api.hackerone.com/docs/v1#/reports

Reply:
Great, so we can merge